tests/Util/Database/StaticDatabase.php

```diff
@@ -179,7 +180,8 @@ public static function statConnect(array $server)
 			self::$staticConnection = @new ExtendedPDO($connect, $user, $pass);
 			self::$staticConnection->setAttribute(PDO::ATTR_AUTOCOMMIT,0);
 		} catch (PDOException $e) {
-			/// @TODO At least log exception, don't ignore it!
+			// Log exception
+			Logger::error('Cannot establish database connection', ['exception' => $e, 'host' => $server, 'dbname' => $db]);
```

Comment:
Plz no dependency injection directly without using the constructor, because that's where the party gets started :)

`Logger::notice()` is calling `self::logger->notice()`, which has to be initialized with proper `LoggerInterface $logger` which is created by a factory using... the `Database` itself :)) and `StaticDatabase` must be called before any DB call, otherwise there may be a transaction mismatching during tests

Reply:
If we could make `Database::$logger` static it would solve this?

Reply:
There is no easy possibility to add a logger here, that's why there isn't any yet and instead a placeholder.. This execution point is a critical state during a testrun, and tbh I'd like to leave here no further logic (yet) because I spent hours debugging cases, where transactions weren't fully closed and strange/unpredictable errors occur (sometimes -mainly during debugging other errors :) ...)

Reply:
Okay, so you suggest me to revert my changes here?

Reply:
Yes please!